sql/index_selection.go

```diff
@@ -380,7 +380,7 @@ func (v *indexInfo) analyzeOrdering(
 	// Analyze the ordering provided by the index (either forward or reverse).
 	fwdIndexOrdering := scan.computeOrdering(v.index, v.exactPrefix, false)
 	revIndexOrdering := scan.computeOrdering(v.index, v.exactPrefix, true)
-	fwdMatch, orderCols := analyzeOrdering(fwdIndexOrdering)
+	fwdMatch, _ := analyzeOrdering(fwdIndexOrdering) // TODO(radu): why is the second return being ignored?
```

Comment:
the `orderCols` return indicates how many total columns a desired ordering has so it should be independent of the argument